src/reducer.js

```diff
@@ -4,11 +4,13 @@ import filter from './reducers/filter';
 import lessons from './reducers/lesson';
 import isStudentMode from './reducers/mode';
 import language from './reducers/language';
+import teacherInfo from './reducers/teacherInfo';
 
 export default combineReducers({
   lessons,
   context,
   filter,
   isStudentMode,
-  language
+  language,
+  teacherInfo
```

Comment:
Det jeg mener er som følger (i TeacherInfoBox.js):
Legg til linjene
```
import {getTeacherInfo} from '../../util';
const teacherInfoContext = require.context('onlyFrontmatter!lessonSrc/', false, /index\.md/);
const teacherInfo = getTeacherInfo(teacherInfoContext);
```
på toppen.

Så kan du i render lenger ned bare bruke `teacherInfo.teacher` og `teacherInfo.assistant` i stedet for `this.props.infoTeacher` og `this.props.infoAssistant` som du gjør nå.

Til slutt kan du fjerne alt som har med teacherInfo i state-treet, og alt blir så meget enklere :)

Dette gitt at vi skal ha infoen liggende i index.md i oppgaver. Dersom det skal hardkodes i codeclub-viewer blir det jo enda enklere, da kan du jo bare lage teksten i et object i en egen js-fil og exportere denne som teacherInfo, og så importere denne js-filen i teacherInfoBox.js.